_events/sr2023/horsham-tech-day-february.md

```diff
@@ -16,7 +16,7 @@ as volunteers able to help you with your kits and hands-on guidance with your
 robots.
 
 Spaces are limited to four teams, so be sure to [let us know][tech-day-signup]
-by Friday 27th January 2023 if you'd like to attend.
```

Comment:
Issue: The 24th is a Tuesday. 27th was correct.

Reply:
Nothing to see here :eyes:
![image](https://user-images.githubusercontent.com/1496834/210651640-b5d3d29e-a212-47c9-a644-ca0c204a61b5.png)